resources/pygment-github.scss

```diff
@@ -0,0 +1,147 @@
+        .kc { color: #79c0ff } /* Keyword.Constant */
+        .kd { color: #ff7b72 } /* Keyword.Declaration */
+        .kn { color: #ff7b72 } /* Keyword.Namespace */
+        .kp { color: #79c0ff } /* Keyword.Pseudo */
+        .kr { color: #ff7b72 } /* Keyword.Reserved */
+        .kt { color: #ff7b72 } /* Keyword.Type */
+        .ld { color: #79c0ff } /* Literal.Date */
+        .m { color: #a5d6ff } /* Literal.Number */
+        .s { color: #a5d6ff } /* Literal.String */
+        .na { color: #c9d1d9 } /* Name.Attribute */
+        .nb { color: #c9d1d9 } /* Name.Builtin */
+        .nc { color: #f0883e; font-weight: bold } /* Name.Class */
+        .no { color: #79c0ff; font-weight: bold } /* Name.Constant */
+        .nd { color: #d2a8ff; font-weight: bold } /* Name.Decorator */
+        .ni { color: #ffa657 } /* Name.Entity */
+        .ne { color: #f0883e; font-weight: bold } /* Name.Exception */
+        .nf { color: #d2a8ff; font-weight: bold } /* Name.Function */
+        .nl { color: #79c0ff; font-weight: bold } /* Name.Label */
+        .nn { color: #ff7b72 } /* Name.Namespace */
+        .nx { color: #c9d1d9 } /* Name.Other */
+        .py { color: #79c0ff } /* Name.Property */
+        .nt { color: #7ee787 } /* Name.Tag */
+        .nv { color: #79c0ff } /* Name.Variable */
+        .ow { color: #ff7b72; font-weight: bold } /* Operator.Word */
+        .pm { color: #c9d1d9 } /* Punctuation.Marker */
+        .w { color: #6e7681 } /* Text.Whitespace */
+        .mb { color: #a5d6ff } /* Literal.Number.Bin */
+        .mf { color: #a5d6ff } /* Literal.Number.Float */
+        .mh { color: #a5d6ff } /* Literal.Number.Hex */
+        .mi { color: #a5d6ff } /* Literal.Number.Integer */
+        .mo { color: #a5d6ff } /* Literal.Number.Oct */
+        .sa { color: #79c0ff } /* Literal.String.Affix */
+        .sb { color: #a5d6ff } /* Literal.String.Backtick */
+        .sc { color: #a5d6ff } /* Literal.String.Char */
+        .dl { color: #79c0ff } /* Literal.String.Delimiter */
+        .sd { color: #a5d6ff } /* Literal.String.Doc */
+        .s2 { color: #a5d6ff } /* Literal.String.Double */
+        .se { color: #79c0ff } /* Literal.String.Escape */
+        .sh { color: #79c0ff } /* Literal.String.Heredoc */
+        .si { color: #a5d6ff } /* Literal.String.Interpol */
+        .sx { color: #a5d6ff } /* Literal.String.Other */
+        .sr { color: #79c0ff } /* Literal.String.Regex */
+        .s1 { color: #a5d6ff } /* Literal.String.Single */
+        .ss { color: #a5d6ff } /* Literal.String.Symbol */
+        .bp { color: #c9d1d9 } /* Name.Builtin.Pseudo */
+        .fm { color: #d2a8ff; font-weight: bold } /* Name.Function.Magic */
+        .vc { color: #79c0ff } /* Name.Variable.Class */
+        .vg { color: #79c0ff } /* Name.Variable.Global */
+        .vi { color: #79c0ff } /* Name.Variable.Instance */
+        .vm { color: #79c0ff } /* Name.Variable.Magic */
+        .il { color: #a5d6ff } /* Literal.Number.Integer.Long */
```

Comment:
Is this still the `github` theme? Maybe you should have two theme files and `@import $pygment_theme` or something.

Reply:
This file contains both `github` and `github-dark`. I didn't figure out how to use 2 theme files, because scss got in the way:

```
Error: Expected string.
  ╷
2 │ @import $thing;
  │         ^
  ╵
```

```
Error: This at-rule is not allowed here.
  ╷
2 │ @if $is_light_theme { @import "math"; }
  │                       ^^^^^^^^^^^^^^
  ╵
```